webrtc.html

```diff
@@ -1750,21 +1750,29 @@ <h4>Set the RTCSessionDescription</h4>
                             object, <var>transceiver</var>, to represent the <a>media
                             description</a>.</p>
                           </li>
+                          <li>
+                            <p>If the <var>description</var> is of type <code>"offer"</code>
+                            and contains a request to receive simulcast, use the order
+                            of the rid values specified in the simulcast attribute to create
+                            an <code><a>RTCRtpEncodingParameters</a></code> dictionary for
+                            each of the simulcast layers, populating the <code>rid</code> member
+                            according to the corresponding rid value, and let <var>sendEncodings</var>
+                            be the list containing the created dictionaries. Otherwise,
```

Comment:
Suggest adding:
Let "supportedEncodings" be the maximum number of encodings that the implementation can support.
If "supportedEncodings" is less than the length of sendEncodings, truncate sendEncodings so that its length is "supportedEncodings".

Reply:
Good idea.